Test_2.scala:2: error: aborted
  Macros.abort
         ^
1 error
